tests/facilitator.sh

```diff
@@ -0,0 +1,129 @@
+}
+
+# fails silently if the command executes successfully.
+function facilitator_init_no_mosaicconfig_fail {
+ fail_silent "./facilitator init --chain-id $auxChainId --chain-id $auxChainId --origin-password '123' --auxiliary-password '123' --origin-rpc 'https://localhost.com:8545' --auxiliary-rpc 'https://localhost.com:8645'"
+}
+
+# fails silently if the command executes successfully.
+function facilitator_init_no_chainid_fail {
+ fail_silent "./facilitator init --mosaic-config $mosaicConfigPath  --origin-password '123' --auxiliary-password '123' --origin-rpc 'https://localhost.com:8545' --auxiliary-rpc 'https://localhost.com:8645'"
+}
+
+# fails silently if the command executes successfully.
+function facilitator_init_no_originpassword_fail {
+ fail_silent "./facilitator init --mosaic-config $mosaicConfigPath --chain-id $auxChainId --auxiliary-password '123' --origin-rpc 'https://localhost.com:8545' --auxiliary-rpc 'https://localhost.com:8645'"
+}
+
+# fails silently if the command executes successfully.
+function facilitator_init_no_auxiliarypassword_fail {
+ fail_silent "./facilitator init --mosaic-config $mosaicConfigPath --chain-id $auxChainId --origin-password '123' --origin-rpc 'https://localhost.com:8545' --auxiliary-rpc 'https://localhost.com:8645'"
+}
+
+# fails silently if the command executes successfully.
+function facilitator_init_no_originrpc_fail {
+ fail_silent "./facilitator init --mosaic-config $mosaicConfigPath --chain-id $auxChainId --origin-password '123' --auxiliary-password '123' --auxiliary-rpc 'https://localhost.com:8645'"
+}
+
+# fails silently if the command executes successfully.
+function facilitator_init_no_auxiliaryrpc_fail {
+ fail_silent "./facilitator init --mosaic-config $mosaicConfigPath --chain-id $auxChainId --origin-password '123' --auxiliary-password '123' --origin-rpc 'https://localhost.com:8545'"
+}
+
+# creates the facilitator-config.json forcefully
+function facilitator_init_force_pass {
+ info "creating facilitator init with --force option"
+ try_silent "./facilitator init --mosaic-config $mosaicConfigPath --chain-id $auxChainId --origin-password '123' --auxiliary-password '123' --origin-rpc 'https://localhost.com:8545' --auxiliary-rpc 'https://localhost.com:8645' -f"
+}
+
+function facilitator_config_present {
+ if [ -s $facilitatorConfigPath ]
+ then
+ 	info "facilitator-config present"
+  else
+    info "facilitator-config does not exists"
+ fi
+}
+
+function remove_facilitator_config {
+ info "removing facilitator-config"
+ rm $facilitatorConfigPath
+
+}
+
+# facilitator init
+facilitator_init_pass
+
+facilitator_config_present
+
+# Above command has already generated facilitator config. So below command should silently fail as it is already present.
+facilitator_init_fail
+
+remove_facilitator_config
+
+facilitator_config_present
```

Comment:
Assert the values, if present or not.

Reply:
Renamed `facilitator_config_present` to `facilitator_config_exists` which will assert if config is not generated.